src/clj/game/core/commands.clj

```diff
@@ -428,6 +429,12 @@
                                          :choices {:card (fn [t] (same-side? (:side t) %2))}}
                                         (map->Card {:title "/rfg command"}) nil)
         "/roll"       #(command-roll %1 %2 value)
+        "/sabotage"   #(when (= %2 :runner)
+                          (resolve-ability
+                            %1 %2
+                            {:async true
+                             :effect (effect (continue-ability (sabotage-ability (constrain-value value 0 1000)) nil nil))}
```

Comment:
This whole ability map could be `(sabotage-ability (constrain-value value 0 1000))`. No need to wrap in the ability map and continue-ability.